README.md

```diff
@@ -74,11 +68,9 @@ the "--" characters and are directly passed to the compiler in the order specifi
   ]
 ```
 
-When an array of flags is passed, the input files should not be specified via the build tools, but
-rather as compilation flags directly.
+- Input files should not be specified via build tools when passing an array of option flags. Instead, you should use compilation flags as you would, directly.
```

Comment:
This was reworded so now the primary indicator (Array of flags) is much later in the sentence. This is an unnecessary change.